Project/Project/Pages/Index.cshtml.cs

```diff
@@ -35,9 +35,24 @@ public IActionResult OnGet(string query)
             {
                 using (var webClient = new WebClient())
                 {
-                    string jsonString = webClient.DownloadString("https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json");
+                    
+                    string countryString = webClient.DownloadString("https://pkgstore.datahub.io/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json");
```

Comment:
Changed the variable name from jsonString to countryString for better understanding.